src/pass/vectorize_loop.cc

```diff
@@ -257,11 +257,27 @@ class Vectorizer : public IRMutator {
           {cond, t, f}, op->call_type, op->func, op->value_index);
     }
   }
+  Expr MutateErf_(const Call* op, const Expr& e) {
```

Comment:
Why do we need this code?

Reply:
If you vectorize erf intrinsic, it'll lead to error in codegen because there's no vectorized intrinsic in the llvm compiler.

Reply:
let us consider use a whitelist approach about what intrinsic can be vectorized

Reply:
Yes, this is also applicable for e.g. prefetching.

Reply:
That's a good idea. I will prepare this whitelist in the followup commits, probably for the vectorization first. We can expand this to more passes in later pr.